mcs/class/referencesource/mscorlib/system/runtime/interopservices/ucomitypeinfo.cs

```diff
@@ -212,6 +212,7 @@ public struct DISPPARAMS
         public int cNamedArgs;
     }
 
+#if !FULL_AOT_RUNTIME
     [Obsolete("Use System.Runtime.InteropServices.ComTypes.EXCEPINFO instead. http://go.microsoft.com/fwlink/?linkid=14202", false)]
```

Comment:
The interface has more dependencies (UCOMITypeComp, UCOMITypeLib) which are not available and I don't want to bring them all